tool/mapeditor.re

```diff
@@ -70,6 +70,8 @@ fn create_map_editor_menu() -> Ui {
                         };
                         MAP_EDITOR_LABEL.text = "Stop Map Editor";
                         add_component(MAP_EDITOR_COMPONENT);
+                        add_component(MOVEMENT_COMPONENT);
+                        MOVEMENT_STATE.enable_fly = true;
```

Comment:
By default flying should not be enabled. The default should be loading a map to play it, in which case you don't want to have flying enabled by default.